app/javascript/application.js

```diff
@@ -1,3 +1,7 @@
 // Entry point for the build script in your package.json
-import "@hotwired/turbo-rails"
-import "./controllers"
+import '@hotwired/turbo-rails';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```

Comment:
- [ ] The app is not starting with the `./bin/dev` command since we are running both apps at the same time. When i run `rails s` which is not recommended, here is what we have in our browser and terminal
![Screenshot from 2023-07-10 22-39-27](https://github.com/dicodiaz/hello-rails-react/assets/46388113/d7b5ed7b-4d24-47de-858e-257b80aeeff7)
![Screenshot from 2023-07-10 22-38-44](https://github.com/dicodiaz/hello-rails-react/assets/46388113/a306f246-9cb3-4fbe-945f-780b678b1ba4)